lib/listen/event/loop.rb

```diff
@@ -6,97 +6,86 @@
 module Listen
   module Event
     class Loop
+      include Listen::FSM
+
       class Error < RuntimeError
-        class NotStarted < Error
-        end
+        class ThreadFailedToStart < Error; end
```

Comment:
Since `NotStarted` is a public interface, please reuse it instead of introducing `ThreadFailedToStart`.

Reply:
I restored it to its old name, `NotStarted`. And if `start` is called when we're already started, I put the code back as it was to simply return as a no-op, since there could be callers who were counting on that public behavior.
https://github.com/guard/listen/pull/482/commits/0b236c1ab903c8653626e8d6404fd1c680238aa8